x-pack/plugins/apm/public/context/UrlParamsContext/helpers.ts

```diff
@@ -48,15 +48,15 @@ export function toBoolean(value?: string) {
   return value === 'true';
 }
 
-export function getPathAsArray(pathname: string) {
+export function getPathAsArray(pathname: string = '') {
   return compact(pathname.split('/'));
 }
 
 export function removeUndefinedProps<T>(obj: T): Partial<T> {
   return pick(obj, value => value !== undefined);
 }
 
-export function getPathParams(pathname: string) {
+export function getPathParams(pathname: string = '') {
```

Comment:
Is this needed? `pathname` should always be defined afaict.

Reply:
The useLocation hook has a default value of {} so in tests you have to do a lot of setup to avoid this function blowing up even if you don’t really need url param values. I thought it was better to just be tolerant here because it shouldn’t really mask an error but there may be other ways to solve…

Reply:
Ah, okay. Perhaps we can get away with just adding it to `getPathParams` and not also `getPathAsArray`?

Reply:
Actually, nvm. Doesn't really matter :)